beacon_node/rest_api/src/beacon.rs

```diff
@@ -103,30 +96,26 @@ pub fn get_block<T: BeaconChainTypes>(
         ))
     })?;
 
-    let response = BlockResponse {
+    Ok(BlockResponse {
         root: block_root,
         beacon_block: block,
-    };
-
-    ResponseBuilder::new(&req)?.body(&response)
+    })
 }
 
 /// HTTP handler to return a `SignedBeaconBlock` root at a given `slot`.
 pub fn get_block_root<T: BeaconChainTypes>(
-    req: Request<Body>,
-    beacon_chain: Arc<BeaconChain<T>>,
-) -> ApiResult {
+    req: Request<Vec<u8>>,
+    ctx: Arc<Context<T>>,
+) -> Result<Hash256, ApiError> {
```

Comment:
Rust books recommend having `pub type Result<T> = std::result::Result<T, ApiError>` alias at the top of a module and use something like `Result<Hash256>` as the function return type.  It's easier to use and there's less repetition because error types tend to stay the same in a single module.